src/activities/multiplication-chart/MultiplicationChart.qml

```diff
@@ -0,0 +1,287 @@
+/* GCompris - multiplication-chart.qml
+ *
+ * Copyright (C) 2016 Varun Kumar <varun13169@iiitd.ac.in>
+ *
+ * Authors:
+ *
+ *   Varun Kumar <varun13169@iiitd.ac.in>
+ *
+ *   This program is free software; you can redistribute it and/or modify
+ *   it under the terms of the GNU General Public License as published by
+ *   the Free Software Foundation; either version 3 of the License, or
+ *   (at your option) any later version.
+ *
+ *   This program is distributed in the hope that it will be useful,
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *   GNU General Public License for more details.
+ *
+ *   You should have received a copy of the GNU General Public License
+ *   along with this program; if not, see <http://www.gnu.org/licenses/>.
+ */
+import QtQuick 2.1
+
+import "../../core"
+import "multiplication-chart.js" as Activity
+
+ActivityBase {
+    id: activity
+
+    onStart: focus = true
+    onStop: {}
+
+    pageComponent: Image {
+        id: background
+        fillMode: Image.PreserveAspectCrop
+        source: "./resource/numbers.svg"
```

Comment:
you should use the qrc path ("qrc:/gcompris/src/activities/{activity}/resource/").